Examples/example_asyncio_class.py

```diff
@@ -68,8 +70,14 @@ async def main():
                                  reverse=False,
                                  range_mode=RotaryIRQ.RANGE_WRAP)
 
-    rotary_encoder_2 = RotaryIRQ(pin_num_clk=32,
-                                 pin_num_dt=33,
+    if sys.platform == 'rp2':
+        encoder2_pin_num_clk = 16
+        encoder2_pin_num_dt = 17
+    else:
+        encoder2_pin_num_clk = 32
+        encoder2_pin_num_dt = 33
+    rotary_encoder_2 = RotaryIRQ(pin_num_clk=encoder2_pin_num_clk,
+                                 pin_num_dt=encoder2_pin_num_dt,
```

Comment:
I think it would be best to remove the platform if/else.  just have pin_num_clk=18 and pin_num_dt=19 in the constructor.
note:  while you are editing, you might consider changing to pin_num_clk=13 and pin_num_dt=14 in the code above (`rotary_encoder_1 = ... `) and also in the other two examples... Or, I can make these changes after the PR is merged as these changes don't relate to your PR.

Reply:
Agreed; I will update the PR code accordingly.  I like that approach; it keeps the code simpler.